kubernetes/podcreation.yml

```diff
@@ -0,0 +1,15 @@
+---
+
+apiVersion: v1
+kind: Pod
+metadata:
+        name: phoenixapp
+        labels:
+                app: phoenixapp
+spec:
+        containers:
+                - name: appcontainer
+                  image: mehmuud/phoenixapp:phoenix
```

Comment:
Is there anything different about the image you created?

We already have a docker image on docker hub `9inpachi/phoenix`. I would recommend you to replace it with that because we update it with every release so this one will become stale quickly.